src/solver/conv_hip_implicit_gemm_3d_fwd_nonpack_xdlops.cpp

```diff
@@ -0,0 +1,373 @@
+/*******************************************************************************
```

Comment:
@iq136boy : Instead of adding a new solver for non-packed case, why not modify the existing 3d grouped solvers for `fwd,bwd,wrw` to use `strides` like you're doing below. That way we get all 3 for both packed and non-packed cases. CC: @JehandadKhan, do you agree?

Reply:
Yeah I agree, with the new solver just for non-packed tensors would result in a lot of duplicate code. More importantly, the underlying kernels are the same since they dont have any assumptions no the tensor strides.

Reply:
> @iq136boy : Instead of adding a new solver for non-packed case, why not modify the existing 3d grouped solvers for `fwd,bwd,wrw` to use `strides` like you're doing below. That way we get all 3 for both packed and non-packed cases. CC: @JehandadKhan, do you agree?

@amberhassaan I agree with it. Currently working on it.